electron-app/package.json

```diff
@@ -90,7 +90,7 @@
     "rimraf": "^2.7.1",
     "wdio-chromedriver-service": "^6.0.4",
     "webdriverio": "^6.10.2",
-    "@theia/electron": "1.10.0"
+    "@theia/electron": "1.11.0"
```

Comment:
Would you do me a favour and move this dependency close to the top of the block, just below `@theia/cli`?